freud/common.pyx

```diff
@@ -24,15 +24,12 @@ cdef class Compute:
     .. code-block:: python
         class Cluster(Compute):
 
-            @Compute._compute("computeClusters")
-            def computeClusters(...)
+            def compute(...)
                 ...
 
-            @Compute._computed_property("computeClusters")
```

Comment:
I'm not sure if you meant to remove this line. I think it should just have no argument?